tools/jenkins/run_distribution.sh

```diff
@@ -134,12 +124,11 @@ elif [ "$platform" == "macos" ]; then
         ;;
     esac
 
-    # clean up
-    cd ~/ 
+    # Clean up
     rm -rf $brew_root
 
     # Make sure the system brew installation is still unaffected
-    export PATH=$OLD_PATH
+    export PATH=$ORIGINAL_PATH
```

Comment:
ah, get it, makes sense. But you actually don't do any check, you just list the installed packages, so perhaps you should clarify the comment.